documentation.html

```diff
@@ -375,6 +373,59 @@ <h4 class="component-heading">Badges</h4>
             loading="lazy"
           >
           </iframe>
+          <p class="component-sub-heading-bold">Example of Badge on text</p>
+          <p class="component-text">
+            We have multiple sizes of text-badges for using these you can use
+            <span class="highlight-text"
+              >text-badge-large, text-badge-medium-large, text-badge-medium,
+              text-badge-small, text-badge-medium-very-small</span
+            >
+            classes.
+          </p>
+          <div class="component-example flex-column flex-gap-1">
+            <p class="font-large weight-semi-bold primary-text-color">
+              Example of large text badge
+              <span
+                class="text-badge text-badge-rounded-border text-badge-large"
+                >New</span
+              >
+            </p>
+            <p class="font-medium-large weight-semi-bold primary-text-color">
+              Example of medium large text badge
+              <span
+                class="text-badge text-badge-rounded-border text-badge-medium-large"
+                >New</span
+              >
+            </p>
+            <p class="font-medium weight-semi-bold primary-text-color">
+              Example of medium text badge
+              <span
+                class="text-badge text-badge-rounded-border text-badge-medium"
+                >New</span
+              >
+            </p>
+            <p class="font-small weight-semi-bold primary-text-color">
```

Comment:
here too font-small can be named as font-sm

Reply:
Yes i will rename